Ci cd is failing

Run # Get current version from TOC
Current version: 6.3.1
New version: 6 3 2
[main dbe003d] chore: bump version to 6 3 2 [skip ci]
 1 file changed, 1 insertion(+), 1 deletion(-)
fatal: 'v6 3 2' is not a valid tag name.
Error: Process completed with exit code 128.

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -29,11 +29,11 @@ jobs:
 
           # Increment patch version
           # This handles x.y.z format
-          NEW_VERSION=$(echo $VERSION | awk -F. '{$NF = $NF + 1;} OFS="." {print $0}')
+          NEW_VERSION=$(echo $VERSION | awk -F. -v OFS=. '{$NF = $NF + 1; print $0}')
           echo "New version: $NEW_VERSION"
 
           # Update TOC file
-          sed -i "s/## Version: $VERSION/## Version: $NEW_VERSION/" AutoDungeonWaypoint.toc
+          sed -i "s/^## Version: .*/## Version: $NEW_VERSION/" AutoDungeonWaypoint.toc
 
           # Configure git
           git config user.name "github-actions[bot]"
@@ -55,4 +55,4 @@ jobs:
       - name: Push changes
         if: github.event_name == 'push' && github.ref == 'refs/heads/main'
         run: |
-          git push origin main --tags
+          git push origin HEAD:main --tags

